### ISSUE TITLE
Error in npm run setup.

### ISSUE THREAD

**Issue by slymnozdmrc** [coryhouse/react-slingshot#570]
I completed initial machine setup than when I run the npm run setup or npm install command, I  got following error:

 node@8.9.4 preinstall C:\Users\sulooz\Desktop\ExampleProject\node_modules\.staging\node-8e5a4863
> node installArchSpecificPackage

```
module.js:549
    throw err;
    ^
Error: Cannot find module 'node-bin-setup'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (C:\Users\sulooz\Desktop\ExampleProject\node_modules\.staging\node-8e5a4863\installArchSpecificPackage.js:1:63)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node@8.9.4 preinstall: `node installArchSpecificPackage`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the node@8.9.4 preinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sulooz\AppData\Roaming\npm-cache\_logs\2018-05-17T07_23_51_333Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-slingshot@7.0.1 setup: `node tools/setup/setupMessage.js && npm install && node tools/setup/setup.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-slingshot@7.0.1 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sulooz\AppData\Roaming\npm-cache\_logs\2018-05-17T07_23_51_523Z-debug.log
```
**Device Information:**
 - OS: Windows 7
 - Node Version: 8.9.4
 - Npm  Version: 4.6.1
 - Editor VS Code

**Comment by coryhouse** [comment]
That's odd. Please try this:
1. Delete package-lock.json
2. Delete the node_modules folder
3. `npm install`
4. `npm start`

**Comment by slymnozdmrc** [comment]
I tried but still got same error. But i run the "npm install node-bin-setup" command than npm install,
I got following error. 
```
`npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.2 (node_modules\watchpack\node_modules\chokidar\node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.4: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})
npm WARN react-router-redux@5.0.0-alpha.6 requires a peer of react@^15 but none was installed.
npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none was installed.
npm WARN babel-loader@7.1.1 requires a peer of webpack@2 || 3 but none was installed.
npm ERR! path C:\Users\sulooz\Desktop\ExampleProject\node_modules\.staging\node-8e5a4863
npm ERR! code EBUSY
npm ERR! errno -4082
npm ERR! syscall rename
npm ERR! EBUSY: resource busy or locked, rename 'C:\Users\sulooz\Desktop\ExampleProject\node_modules\.staging\node-8e5a4863' -> 'C:\Users\sulooz\Desktop\ExampleProject\node_modules\node'

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sulooz\AppData\Roaming\npm-cache\_logs\2018-05-17T11_45_22_428Z-debug.log`
```

**Comment by slymnozdmrc** [comment]
I tried the same operation on my macbook I didn't get any error. Because of this, I decided to use same nodejs and npm version on my windows. So that problem solved also on my windows. I follow the below steps for solving problem. 
 
1. change nodejs version with 9.10.1 
2. upgrade npm version to 5.6.0
3. delete node_modules
4. delete package-lock.json
5. `npm install`
6. `npm start -s`

Thanks for help this is very nice starter kit :+1:
